jujugui/static/gui/src/app/app.js

```diff
@@ -703,42 +802,48 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Handles logging into both the env and controller api websockets.
+
+      @method loginToAPIs
+      @param {Object} credentials The credentials to pass to setCredentials.
+      @param {Boolean} useMacaroons If it should use the loginWithMacaroon
+        login methods.
+    */
+    loginToAPIs: function(
+      credentials, useMacaroons, apis=[this.env, this.controllerAPI]) {
+      if (useMacaroons) {
+        apis.forEach(api => {
+          api.loginWithMacaroon(new Y.juju.environments.web.Bakery({
+            webhandler: new Y.juju.environments.web.WebHandler(),
+            interactive: true,
+            serviceName: 'juju',
+            dischargeStore: window.localStorage
+          }));
+        });
```

Comment:
Add a return here and drop the else block, therefore avoiding one level of indentation for the rest of the method body.